AAUnlimited/General/DrawD3D.cpp

```diff
@@ -477,6 +477,12 @@ namespace DrawD3D {
 			General::utf8.from_bytes(Subtitles::fontFamily).c_str(),
 			&Subtitles::Font, false, "Subs Font creation failed");
 
+		// Notifications Font
+		CreateFontD3d(Notifications::fontSize, 0, FW_ULTRABOLD, 1, false, DEFAULT_CHARSET,
+			OUT_DEFAULT_PRECIS, DEFAULT_QUALITY, DEFAULT_PITCH | FF_DONTCARE,
+			General::utf8.from_bytes(Notifications::fontFamily).c_str(),
+			&Notifications::Font, false, "Notifications Font creation failed");
```

Comment:
What do we need different fonts everywhere for? Please use the same font for everything.

Reply:
For many people it is more convenient, for example, when the text of the subtitles is larger than the text of the notifications.